sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj

```diff
@@ -1,59 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <AssemblyTitle>Microsoft.Azure.EventHubs</AssemblyTitle>
     <Description>This is the next generation Azure Event Hubs .NET Standard client library. For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>3.0.0</Version>
-    <AssemblyVersion>3.0.0.0</AssemblyVersion>
-    <FileVersion>3.0.0.0</FileVersion>
     <VersionPrefix>3.0.0</VersionPrefix>
-    <PackageId>Microsoft.Azure.EventHubs</PackageId>
     <PackageTags>Azure;Event Hubs;EventHubs;.NET;AMQP;IoT</PackageTags>
     <PackageReleaseNotes>https://github.com/Azure/azure-event-hubs-dotnet/releases</PackageReleaseNotes>
     <DocumentationFile>$(OutputPath)$(TargetFramework)\Microsoft.Azure.EventHubs.xml</DocumentationFile>
-  </PropertyGroup>
-
-  <PropertyGroup>
```

Comment:
This needs to stay;  we're overriding the general target frameworks as part of the external build properties.  `NetStandard1.4` isn't in the base set and is only needed for this assembly in theEvent Hubs ecosystem.

Reply:
The comment alignment didn't come out well, but this is the block that I'm talking about:

```xml
<PropertyGroup>
    <TargetFrameworks>$(RequiredTargetFrameworks);netstandard1.4</TargetFrameworks>
</PropertyGroup>
```

Reply:
That is still there I just merged it with the PropertyGroup directly above it.

Reply:
ahhh...   my bad.  I missed that in the diff.